### PR TITLE
Adds support for inClusterConfig

### PR DESCRIPTION
**Pull request by abhide**

### CODE REVIEW

**Comment by abhide** [comment]
@gotwarlost can you take a look? Will add more force options in subsequent PRs

File: setup.go [hunk 1]
```diff
@@ -130,9 +130,11 @@ func setup(root *cobra.Command) {
 	var cp commands.ConfigFactory
 	var rootDir string
 	var appTag string
+	var forceContext string
 
 	vmConfigFn := vm.ConfigFromCommandParams(root, "vm:", true)
 	cfg := remote.NewConfig(root, "k8s:")
+	root.PersistentFlags().StringVar(&forceContext, "force:k8s-context", "__current__", "Force K8s context")
```
Comment (gotwarlost) [inline setup.go]: default should `""` - we want to use the env's derived context by default

File: internal/commands/config.go [hunk 2]
```diff
@@ -124,11 +130,14 @@ func (cp ConfigFactory) internalConfig(app *model.App, vmConfig vm.Config, clp c
 }
 
 // Config returns the command configuration.
-func (cp ConfigFactory) Config(app *model.App, vmConfig vm.Config, remoteConfig *remote.Config) (*Config, error) {
+func (cp ConfigFactory) Config(app *model.App, vmConfig vm.Config, remoteConfig *remote.Config, forceContext string) (*Config, error) {
```
Comment (gotwarlost) [inline internal/commands/config.go]: why not just pass `forceOptions` here

File: setup.go [hunk 3]
```diff
@@ -130,9 +130,11 @@ func setup(root *cobra.Command) {
 	var cp commands.ConfigFactory
 	var rootDir string
 	var appTag string
+	var forceContext string
```
Comment (gotwarlost) [inline setup.go]: maybe just declare `forceOptions` and bind flag to member?

File: internal/commands/config.go (hunk 2 above)
Comment (abhide) [reply internal/commands/config.go]: I would have make export `forceOptions`. I thought we didn't want to expose members under `internal` outside.

File: setup.go (hunk 1 above)
Comment (abhide) [reply setup.go]: DONE

**Pull request merged**